pbsmetrics/go_metrics.go

```diff
@@ -412,8 +433,10 @@ func (me *Metrics) RecordAdapterBidReceived(labels AdapterLabels, bidType openrt
 	// Adapter metrics
 	am.BidsReceivedMeter.Mark(1)
 	// Account-Adapter metrics
-	aam := me.getAccountMetrics(labels.PubID).adapterMetrics[labels.Adapter]
-	aam.BidsReceivedMeter.Mark(1)
+	if me.accountMetrics != nil {
```

Comment:
This is correct, anything that has both `labels.PubID` and `labels.Adapter` we want to get rid of. We want to keep it when it is just one or the other.

Reply:
With the check implemented inside of `getAccountMetrics(id string)` we are only not setting metrics named like: `account.<labels.PubID>.<ADAPTER_NAME>.<SOME_METIRC>` but in the newest version of the code, we still need a null check on whether or not  `labels.Adapter` is found in the `adapterMetrics` hash table.